lib/livebook/settings.ex

```diff
@@ -213,7 +218,8 @@ defmodule Livebook.Settings do
 
   ## Messages
 
-    * `{:env_vars_changed, env_vars}`
+    * `{:env_var_changed, env_var}`
```

Comment:
```suggestion
    * `{:env_var_set, env_var}`
```

it matches `Settings.set_env_var` and is more clear :)

Reply:
and change the `deleted` to `unset`?

Reply:
Either works for me, but let's do that! I would reflect this naming into the runtime function names, hopefully a global replace will do :D

Reply:
I would keep the runtime function naming coz `System` uses `put` and `delete` for environment variables

Reply:
Fine by me too :)